base/commandLine.cpp

```diff
@@ -1039,8 +1175,15 @@ bool processSettings(Common::String &command, Common::StringMap &settings, Commo
 		return true;
 	} else if (command == "auto-detect") {
 		// If auto-detects succeed, we want to return false so that the game is started
-		return !autoDetect(settings["path"]);
+		return !detectGames(settings["path"], true);
 		//return true;
+	} else if (command == "detect") {
+		detectGames(settings["path"], false);
+		return true;
+	} else if (command == "add") {
+		return !addGame(settings["path"]);
```

Comment:
For context, feature requests are just that: requests. They haven't necessarily been discussed or fully worked out.